src/store/modules/socket/user.ts

```diff
@@ -50,7 +50,7 @@ const SocketUserModule: Module<SocketUserState, RootState> = {
       if (jti) {
         const user = state.users.find((user: User) => user.jti === jti)
         if (user) {
-          return user.roles.find((role: Role) => role.roleTypes === RoleTypes.USER)
+          return user.roles.length
```

Comment:
this can return more than just 1, 
also 1 is not same as true with ===

return (user.length === 1) ? true : false

Reply:
I actually just need to use double bangs. Good spot though